notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1531 @@
+    "     - [display.clear_output](http://ipython.org/ipython-doc/dev/api/generated/IPython.display.html#IPython.display.clear_output) - clear text from cells\n",
+    " - [**numpy**](https://numpy.org) - math and array functions\n",
+    "     - [percentile](https://numpy.org/doc/stable/reference/generated/numpy.percentile.html) - lognorm percentile clipping\n",
+    " - [**matplotlib**](https://matplotlib.org) - create graphics\n",
+    "     - [pyplot](https://matplotlib.org/stable/tutorials/pyplot.html) - make figures and graphics\n",
+    "     - [colors.LogNorm](https://matplotlib.org/stable/api/_as_gen/matplotlib.colors.LogNorm.html) - display image normalization\n",
+    "     - [patches.Rectangle](https://matplotlib.org/stable/api/_as_gen/matplotlib.patches.Rectangle.html) - overlay rectangular regions\n",
+    " - [**astroquery**](https://astroquery.readthedocs.io/en/latest/) - download astronomical data\n",
+    "     - [mast.Observations](https://astroquery.readthedocs.io/en/latest/mast/mast_obsquery.html) - query MAST database\n",
+    " - [**astropy**](https://www.astropy.org) - model fitting and file handling\n",
+    "     - [io.fits](https://docs.astropy.org/en/stable/io/fits/) - import FITS files\n",
+    "     - [table.QTable](https://docs.astropy.org/en/stable/api/astropy.table.QTable.html) - tables with physical units\n",
+    "     - [modeling.models](https://docs.astropy.org/en/stable/modeling/models.html) - Gaussian and Moffat models\n",
+    "     - [modeling.fitting](https://docs.astropy.org/en/stable/modeling/fitting.html) - fitting methods for models\n",
+    "     - [visualization.simple_norm](https://docs.astropy.org/en/stable/api/astropy.visualization.mpl_normalize.simple_norm.html) - display image normalization\n",
+    "     - [stats.sigma_clipped_stats](https://docs.astropy.org/en/stable/api/astropy.stats.sigma_clipped_stats.html) - mean, median, and standard deviation\n",
+    "     - [stats.SigmaClip](https://docs.astropy.org/en/stable/api/astropy.stats.SigmaClip.html) - sigma-clip provided data\n",
+    " - [**photutils**](https://photutils.readthedocs.io/en/stable/index.html) - aperture and PSF photometry tools\n",
+    "     - [detection.DAOStarFinder](https://photutils.readthedocs.io/en/stable/api/photutils.detection.DAOStarFinder.html#photutils.detection.DAOStarFinder) - finding stars in images\n",
+    "     - [detection.find_peaks](https://photutils.readthedocs.io/en/stable/api/photutils.detection.find_peaks.html#photutils.detection.find_peaks) - finding peaks in images\n",
+    "     - [photutils.centroids](https://photutils.readthedocs.io/en/stable/centroids.html) - finding stellar centroids\n",
+    "     - [photutils.aperture](https://photutils.readthedocs.io/en/stable/aperture.html) - aperture photometry tools\n",
+    "       - [aperture_photometry](https://photutils.readthedocs.io/en/stable/aperture.html) - measure aperture photometry\n",
```

Comment:
Correct [link](https://photutils.readthedocs.io/en/stable/api/photutils.aperture.aperture_photometry.html#photutils.aperture.aperture_photometry) for `aperture_photometry()`

Reply:
Corrected!